src/components/views/dialogs/spotlight/SpotlightDialog.tsx

```diff
@@ -351,7 +351,12 @@ const SpotlightDialog: React.FC<IProps> = ({ initialText = "", initialFilter = n
             userIds.add(userId);
             return userIds;
         }, new Set<string>());
-        for (const user of [...findVisibleRoomMembers(cli, msc3946ProcessDynamicPredecessor), ...users]) {
+
+        const lcQuery = trimmedQuery.toLowerCase();
+        const members = findVisibleRoomMembers(cli);
```

Comment:
should probably keep this:
```suggestion
        const members = findVisibleRoomMembers(cli, msc3946ProcessDynamicPredecessor);
```